### DISCUSSION TITLE
ILI9486 and RP2040 problem

### DISCUSSION THREAD

**Discussion by KBLiveSolutions** [moononournation/Arduino_GFX#212] [Q&A]
Hi

I am trying to get my Pico work with [this Waveshare 4'' display](https://www.waveshare.com/4inch-tft-touch-shield.htm)
No matter what I've tried, I only get a white screen

I am using 
`
 Arduino_DataBus *bus = new Arduino_RPiPicoSPI(TFT_DC, TFT_CS, PIN_SPI0_SCK /* SCK */, PIN_SPI0_MOSI /* MOSI */, PIN_SPI0_MISO /* MISO */, spi0 /* spi */);`
`Arduino_GFX *gfx = new Arduino_ILI9486_18bit(bus, TFT_RST, 0 /* rotation */, false /* IPS */);`

and pins

`Vin               -> Vcc`
`GND               -> GND`
`GPIO 17           -> CS`
`GPIO 26           -> RESET`
`GPIO 27           -> D/C`
`GPIO 19(MOSI)     -> SDI(MOSI)`
`GPIO 18(SCK)      -> SCK`
`GPIO 28           -> LED`

but all I get is a white screen.
What can possibly be wrong ?

I am using Earle Philhower port of the Pico.

**Comment by moononournation** [comment]
![image](https://user-images.githubusercontent.com/13228891/193972644-edd917d5-9105-44fe-9d28-c1b3e4fe42e5.jpeg)
Just tested with my old ILI9486L display, it is ok with latest software.

**Comment by moononournation** [comment]
Seems the board in your link is for 5V board, it may not support Pico running in 3.3V.

**Reply by KBLiveSolutions** [reply]
Thanks for your answer.
From what I understand, it seems to be 3.3V tolerant
<img width="1008" alt="Capture d’écran 2022-10-05 à 08 49 10" src="https://user-images.githubusercontent.com/84496075/193998309-6313e2f7-b663-4725-8f36-38c88b5788cd.png">

Do you have a link to a board similar to yours ?
